app/assets/javascripts/app/discussions/index/sidebar.html.slim

```diff
@@ -1,10 +1,11 @@
 md-sidenav.Sidebar.md-sidenav-left.md-whiteframe-z2 md-component-id="left" md-is-open="ctrl.showSidebar"
   md-toolbar.md-warn.md-whiteframe-z1 ng-click="ctrl.toggleSidebar()"
-    .md-toolbar-tools layout="row" layout-align="space-around center"
+    .Button.md-toolbar-tools layout="row" layout-align="space-around center"
```

Comment:
Because the "Close menu", which looks like a button at first sight (it's saying "click me to close the menu"), was not behaving like a button on mouse hover.